worlds/iris_fpv_cam.world

```diff
@@ -28,9 +28,9 @@
           <contact_surface_layer>0.001</contact_surface_layer>
         </constraints>
       </ode>
-      <max_step_size>0.004</max_step_size>
+      <max_step_size>0.002</max_step_size>
       <real_time_factor>1</real_time_factor>
-      <real_time_update_rate>250</real_time_update_rate>
+      <real_time_update_rate>500</real_time_update_rate>
```

Comment:
Is there a reason the `max_step_size` and `real_time_update_rate` are both configurable but needs to correspond to the reciprocal number?

Reply:
If they don't match your simulation runs slower or faster but that's what we want to specify with the `PX4_SIM_SPEED_FACTOR`.

Reply:
Wouldn't the code just error if those two don't match? or maybe I am just confused...

Reply:
Yes, it would. Now I'm confused.

Reply:
@julianoes Ah, okay. My question was why are both of the parameters exposed in the sdf file If we need both of them to match?

Reply:
Aha, I think that's historically.